### PR TITLE
fixes mozilla#9088 feat(nimbus): add targeting to support existing us…

### PR DESCRIPTION
**Pull request by halemu**
Land advanced targeting for existing users with no vpn/enterprise history, and haven't disabled recommended features/extensions

### CODE REVIEW

File: experimenter/experimenter/targeting/constants.py [hunk 1]
```diff
@@ -605,6 +605,23 @@ def __post_init__(self):
     is_first_run_required=False,
     application_choice_names=(Application.DESKTOP.name,),
 )
+EXISTING_USER_NO_VPN_HAS_NOT_DISABLED_RECOMMEND_FEATURES = NimbusTargetingConfig(
+    name="Existing users, no enterprise or past VPN use, hasn't disabled"
+    " 'Recommend extensions/features'",
```
Comment (aminomancer) [inline experimenter/experimenter/targeting/constants.py:610]:
```suggestion
    name=(
        "Existing users, no enterprise or past VPN use, hasn't disabled "
        "'Recommend extensions/features'"
    ),
```

File: experimenter/experimenter/targeting/constants.py [hunk 2]
```diff
@@ -605,6 +605,23 @@ def __post_init__(self):
...
+    slug="existing_user_no_vpn_has_not_disabled_recommend_features",
+    description="Exclude users who have used Mozilla VPN, are enterprise users, or have"
+    " disabled 'Recommend extensions/features'",
```
Comment (aminomancer) [inline experimenter/experimenter/targeting/constants.py:613]:
```suggestion
    description=(
        "Exclude users who have used Mozilla VPN, are enterprise users, or have"
        " disabled 'Recommend extensions/features'"
    ),
```

File: experimenter/experimenter/targeting/constants.py (hunk 1 above)
Comment (brennie) [inline experimenter/experimenter/targeting/constants.py:610]: Could you reformat these strings like you have in targeting:

```
name=(
    "...."
    "...."
)
```

File: experimenter/experimenter/targeting/constants.py [hunk 3]
```diff
@@ -605,6 +605,23 @@ def __post_init__(self):
...
+    targeting=(
+        f"{EXISTING_USER_NO_ENTERPRISE_OR_PAST_VPN.targeting} && "
+        "'browser.newtabpage.activity-stream.asrouter.userprefs.cfr.features'|preferenceValue"
+        "&&"
+        "'browser.newtabpage.activity-stream.asrouter.userprefs.cfr.addons'|preferenceValue"
+    ),
```
Comment (aminomancer) [inline experimenter/experimenter/targeting/constants.py:619]:
```suggestion
    targeting=(
        f"{EXISTING_USER_NO_ENTERPRISE_OR_PAST_VPN.targeting} && "
        "'browser.newtabpage.activity-stream.asrouter.userprefs.cfr.features'|preferenceValue &&"
        "'browser.newtabpage.activity-stream.asrouter.userprefs.cfr.addons'|preferenceValue"
    ),
```

File: experimenter/experimenter/targeting/constants.py (hunk 2 above)
Comment (brennie) [inline experimenter/experimenter/targeting/constants.py:613]: Here too.

**Review by brennie — CHANGES_REQUESTED** [review]

**Review by aminomancer — APPROVED** [review]
Some formatting nits but looks good

**Review by brennie — APPROVED** [review]

**Review by yashikakhurana — APPROVED** [review]
thank you @halemu 🎉

**Pull request merged**